src/main/java/mate/academy/spring/mapper/impl/response/ShoppingCartResponseMapper.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.spring.mapper.impl.response;
+
+import java.util.ArrayList;
+import java.util.List;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.dto.response.ShoppingCartResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ShoppingCartResponseMapper
+        implements DtoResponseMapper<ShoppingCartResponseDto, ShoppingCart> {
+    @Override
+    public ShoppingCartResponseDto toDto(ShoppingCart shoppingCart) {
+        ShoppingCartResponseDto responseDto = new ShoppingCartResponseDto();
+        responseDto.setId(shoppingCart.getId());
+        responseDto.setUserId(shoppingCart.getUser().getId());
+        List<Ticket> ticketList = shoppingCart.getTickets();
```

Comment:
Instead of manually creating a new ArrayList and iterating through ticketList, consider using Java 8 Stream API to map and collect the ticketIds.